index.php

```diff
@@ -166,16 +167,16 @@
 $a->page['end'] = '';
 
 
-if (! x($_SESSION, 'sysmsg')) {
+if (x($_SESSION, 'sysmsg') === false) {
```

Comment:
Why don't you use the ```defaults``` function for this?

Reply:
Fair enough.

Reply:
Done.